src/content/operations/upgrading/_index.en.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Upgrading"
+date: 2023-09-06T14:06:11+02:00
+weight: 15
+---
+
+Starting with Submariner 0.16, the recommended way to upgrade Submariner is via the [`subctl upgrade`](#automated-upgrade) command.
+This can be used to upgrade clusters to Submariner 0.16.
+To upgrade older clusters to a version of Submariner before 0.16, follow the [manual upgrade process](#manual-upgrade).
+
+## Automated Upgrade
+
+If your current version of `subctl` (as indicated by `subctl version`) is older than 0.16,
+start by [installing the desired version of `subctl`](../deployment/subctl/#install).
+If your current version of `subctl` is 0.16 or later, it will upgrade itself during the upgrade process.
+
+Once you have `subctl` 0.16 or later, run it with a kubeconfig pointing to the cluster(s) you wish to upgrade:
+
+```bash
+subctl upgrade --kubeconfig /path/to/kubeconfig
+```
+
+(The `--kubeconfig` parameter is optional; `subctl` will use any configuration that `kubectl` would find.)
+
+`subctl upgrade` will start by upgrading `subctl` to the latest released version,
```

Comment:
Just curious:

What if the kubeconfig has multiple contexts pointing to the same cluster. Does it run upgrade against all the contexts?

Reply:
Yes, the end of the sentence was supposed to convey that but I should clarify.